### PR TITLE
Allow moving selected photos to a different person

### PR DESCRIPTION
**Pull request by marcelklehr**
fixes nextcloud/recognize#602

### CODE REVIEW

File: src/views/FaceContent.vue [hunk 1]
```diff
@@ -348,6 +363,18 @@ export default {
 			}
 		},
 
+		async handleMove(faceName, fileIds) {
+			try {
+				this.loadingCount++
+				await this.moveFilesToFace({ oldFace: this.faceName, faceName, fileIdsToMove: fileIds })
+				this.showMoveModal = false
+			} catch (error) {
+				logger.error(error)
```
Comment (artonge) [inline src/views/FaceContent.vue:372]: And `showError` ?

**Review by artonge — APPROVED** [review]

File: src/views/FaceContent.vue (hunk 1 above)
Comment (marcelklehr) [reply src/views/FaceContent.vue:372]: showError is called in moveFilesToFace

**Pull request merged**